### DISCUSSION TITLE
Add additional filter for all unread threads in current room

### DISCUSSION THREAD

**Discussion by moritzdietz** [element-hq/element-meta#1825] [Ideas]
#### What would you like to do?
In addition to the current selection of "All threads" and "My threads" a third option "Unread threads" where only unread threads are listed would be made available as a user.

#### Why would you like to do it?
This would give me the ability to have a constant view of all unread threads besides a list of all threads including the unread ones. Like some E-Mail clients give you the ability to have a custom view for all unread E-Mails this would be the equivalent.

#### How would you like to achieve it?
Add a third option below these two:
<img width="463" alt="image" src="https://github.com/vector-im/element-web/assets/7935536/1348ae01-7f00-404e-99e6-37b96404265e">

### Have you considered any alternatives?

The alternative is to select "All threads" and hope for Element Web/Desktop to sort unread threads at the bottom so they are easily accessible as there is no way to search in the filter for specific threads.

**Comment by JurgenG** [comment]
Members on our channel or complaining about not being able to find "unread" messages at least every few days. Not sure if these are false positives (not really unread items), but being able to filter on unread threads would most certainly give peace of mind. Also if people respond to an old thread, it becomes really hard to find this.
